test/test_core.py

```diff
@@ -6059,6 +6059,9 @@ def test_unistd_links(self, args, nodefs):
 
     self.do_run_in_out_file_test('unistd/links.c', emcc_args=args)
 
+  def test_unistd_write_broken_link(self):
```

Comment:
Can you add `@also_with_noderawfs` here?  How about `@also_with_wasmfs`?  I guess that later probably doesn't work because we are not feature complete yet?

Reply:
Yes to `@also_with_noderawfs` but no to `@also_with_wasmfs`.